src/ui/public/chrome/api/angular.js

```diff
@@ -26,14 +30,37 @@ module.exports = function (chrome, internals) {
       return a.href;
     }()))
     .config(chrome.$setupXsrfRequestInterceptor)
-    .run(($location) => {
+    .run(($location, $rootScope, Private) => {
       chrome.getFirstPathSegment = () => {
         return $location.path().split('/')[1];
       };
 
       chrome.getBreadcrumbs = () => {
         return $location.path().split('/').slice(1);
       };
+
+      const notify = new Notifier();
+      const urlOverflow = Private(UrlOverflowServiceProvider);
+      const check = (event) => {
+        if ($location.path() === '/error/url-overflow') return;
+
+        try {
+          if (urlOverflow.check($location.absUrl()) <= 150) {
```

Comment:
When you are within 150 characters of the max you get a yellow "warning" notifications. Hopefully preventing you from actually overflowing the url.